ember/app/ui/form-configuration/controller.js

```diff
@@ -0,0 +1,59 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { queryManager } from "ember-apollo-client";
+import { lastValue, restartableTask } from "ember-concurrency-decorators";
+import saveFormMutation from "mysagw/gql/mutations/save-form.graphql";
+import getRootFormsQuery from "mysagw/gql/queries/get-root-forms.graphql";
+
+export default class FormConfigurationController extends Controller {
+  @service notification;
+  @service intl;
+
+  @queryManager apollo;
+
+  @lastValue("fetchForms") forms;
+  @restartableTask
+  *fetchForms() {
+    const forms = yield this.apollo.query(
+      {
+        query: getRootFormsQuery,
+        variables: { isPublished: true, isArchived: false },
+        fetchPolicy: "network-only",
+      },
+      "allForms.edges"
+    );
+
+    return forms.mapBy("node");
+  }
+
+  @restartableTask
+  *setFormMeta(form, formType) {
+    try {
+      const meta = Object.assign({}, form.meta);
+
+      if (formType in meta) {
+        meta[formType] = !meta[formType];
+      } else {
+        meta[formType] = true;
+      }
```

Comment:
Just `meta[formType] = !meta[formType];` should suffice

Reply:
Done

Reply:
Did you forget to push maybe? I don’t see the changes..

Reply:
Pushed with the nav improvement